charts/coredns/coredns.chartserviceversion.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: litmuchaos.io/v1alpha1
+kind: ChartServiceVersion
+metadata:
+  name: coredns
+  version: 0.1.0
+  annotations:
+    categories: Kubernetes
+    chartDescription: Injects coredns chaos 
+spec:
+  displayName: CoreDNS chaos
+  categoryDescription: >
+    Kubernetes is an open-source system for automating deployment, scaling, and management of containerized applications. It groups containers that make up an application into logical units for easy management and discovery. Install will all the experiments which can be used to inject chaos into containerized appications.
+
+  experiments:
+    - pod-network-loss
```

Comment:
should have only coredns-pod-deletee right?

Reply:
done